DATA/SHIPS/shiparch.ini

```diff
@@ -2357,7 +2357,7 @@ ids_info = 461098
 ids_info1 = 461099
 ;res html
 ; \m\bSmall Transport\B\l
-; 

```

Comment:
This and the following removed empty spaces break the infocard of the transport.

Reply:
All of these have been fixed now.